opm/material/densead/Evaluation.hpp

```diff
@@ -73,65 +83,76 @@ class Evaluation
     template <class RhsValueType>
     Evaluation(const RhsValueType& c)
     {
-        value_ = c;
-        std::fill(derivatives_.begin(), derivatives_.end(), 0.0);
+        setValue( c );
+        clearDerivatives();
+        Valgrind::CheckDefined( data_ );
     }
 
-    // create a function evaluation for a "naked" depending variable (i.e., f(x) = x)
+    // create an evaluation which represents a constant function
+    //
+    // i.e., f(x) = c. this implies an evaluation with the given value and all
+    // derivatives being zero.
     template <class RhsValueType>
-    static Evaluation createVariable(const RhsValueType& value, unsigned varPos)
+    Evaluation(const RhsValueType& c, const int varPos)
```

Comment:
the additional constructor. it's new, right? (createVariable() stays around AFAICS.)